### ISSUE TITLE
Price file(s)/base corruption when aborting blocking operations?

### ISSUE THREAD

**Issue by paramecie** [rgriebl/brickstore#300]
I opened an existing old BSX file, for testing.

I applied Average Current Inventory price to all items.
CTRL+A CTRL+G

There was 100s prices to update, and to see what happens, I clicked on the top right button something like "Cancel all blocking operations".
Fine, it stopped.

Since then, even manually item per item, even having closed (without saving) the file, closed BrickStore, I've difficulties to make it apply a price guide price, even if this price exists (see the picture):

![image](https://user-images.githubusercontent.com/32078594/111009463-fb087a00-8393-11eb-9cce-42f183c1c105.png)

Sometimes it works, sometimes not.
There's a price in Price Guide, it doesn't want to apply.

If I force it to redownload, it works.

If I do CTRL+A CTRL+G, it still doesn't want to update 19 prices.
Even if each individual lot does have a valid price guide.

Question: is there a corruption somewhere when you abort a bunch of PG downloads?
In short, it's written somewhere the price doesn't exist (because the abort), and it uses this (wrong) info?

(I tried on another similar file, if I don't abort, zero problem)

**Comment by rgriebl** [comment]
I sadly can't reproduce it here.

> If I do CTRL+A CTRL+G, it still doesn't want to update 19 prices.
> Even if each individual lot does have a valid price guide.

When you do/did this: did BrickStore tell you that it:
1.  update the prices to PG values (although it didn't)
2. there was a problem with <number> of lots
3. there was no message box at all?

**Comment by paramecie** [comment]
I'll try to reproduce it later today.

**Comment by paramecie** [comment]
OK, found it.
1) cleared the "P" cache folder
2) CTRL+I CTRL+I the 10188-1

3) CTRL+A CTRL+G any average

4) cancel downloading shortly after:

![image](https://user-images.githubusercontent.com/32078594/111171521-4cd41e80-85a5-11eb-9843-ce9636c99196.png)

_(which is I wanted, even if the message could simply be "... because you just freaking cancelled it, you brick newbie dumbass!")_

5) For testing, I tried to set average to 2 or such prices, individually, it worked.

6) What doesn't work is if now you do CTRL+A CTRL+G

![image](https://user-images.githubusercontent.com/32078594/111171470-3ded6c00-85a5-11eb-9b0b-1bdd79ee06fd.png)

It doesn't want to download anything.

It seems it's stuck to the previous cancellation?
